proposals/runtime_get_contexts.md

```diff
@@ -103,8 +103,8 @@ extension.ContextType = {
   BACKGROUND: 'BACKGROUND',
   // Offscreen documents for the extension.
   OFFSCREEN_DOCUMENT: 'OFFSCREEN_DOCUMENT',
-  // A side panel context. Currently under development in Chromium.
-  SIDE_PANEL,
+  // A side panel context.
```

Comment:
```suggestion
  // A side panel or sidebar context.
```

While you are at at it - in Firefox we used the side_panel type for the sidebar_action API.

Reply:
Good idea - thanks!